src/main.rs

```diff
@@ -41,12 +47,16 @@ pub fn error(message: &str) -> ! {
 }
 
 /// Runs the benchmark for the given commands
-fn run(commands: &Vec<&str>, options: &HyperfineOptions) -> io::Result<()> {
+fn run(
+    commands: &Vec<&str>,
+    options: &HyperfineOptions,
+    exporter: &mut Option<Box<ExportManager>>,
```

Comment:
Same as above: I would rather like `run` to return the results instead of passing a mutable `ExportManager`.

Reply:
Sure, that'll simplify everything